minecraft/forge/src/main/java/dev/kosmx/playerAnim/impl/neoforge/HelperImpl.java

```diff
@@ -1,24 +1,11 @@
 package dev.kosmx.playerAnim.impl.neoforge;
 
+import net.neoforged.fml.loading.LoadingModList;
 import org.jetbrains.annotations.ApiStatus;
-import org.spongepowered.asm.service.MixinService;
-
-import java.io.IOException;
 
 @ApiStatus.Internal
 public class HelperImpl {
     public static boolean isBendyLibPresent() {
-        return hasClass("io.github.kosmx.bendylib.impl.ICuboid");
-    }
-
-
-    private static boolean hasClass(String name) {
-        try {
-            // This does *not* load the class!
-            MixinService.getService().getBytecodeProvider().getClassNode(name);
-            return true;
-        } catch (ClassNotFoundException | IOException e) {
-            return false;
-        }
+        return LoadingModList.get().getModFileById("bendylib") != null;
```

Comment:
Initially I was trying to get mod from API, but that version of forge simply breaks if I do this before forge is ready.

Is this solution safe to access from early entry?

Reply:
Yes, it's a **Loading**ModList, not a ModList (there are two in FML).

Reply:
Okay, I'll verify if it works correctly and if it does, I'll use this